backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -19,7 +19,7 @@
                 <%= yield :tabs %>
               </div>
             <% end %>
-            <div class="<%= if content_for?(:sidebar) then 'twelve' else 'sixteen' end %> columns">
+            <div id="content-without-nav" class="<%= if content_for?(:sidebar) then 'twelve' else 'sixteen' end %> columns">
```

Comment:
This is an unfortunate name, but `content` has already been taken by the div above that contains both the `aside` nav and the actual content. If anyone comes up with a better name, I'd be happier.